src/coreclr/tools/Common/Compiler/ProcessLinkerXmlBase.cs

```diff
@@ -762,8 +762,7 @@ public class CecilCompatibleTypeParser
         public static TypeDesc? GetType(ModuleDesc assembly, string fullName)
         {
             Debug.Assert(!string.IsNullOrEmpty(fullName));
-            var position = fullName.IndexOf('/');
-            if (position > 0)
```

Comment:
Not sure if it was excluding first char on purpose

Reply:
We don't decide the rules for this. This code mirrors how Cecil parses the string: https://github.com/dotnet/cecil/blob/a994a9fa0165af616cf21b3d418d70387c4bd8ff/Mono.Cecil/ModuleDefinition.cs#L712-L714

This change would make us start treating strings starting with '/` differently.

This change should be reverted, we want to be bug for bug compatible with Cecil and accept whatever weird thing Cecil accepts since we made that our file format for better or worse (I'm certain it is the latter).